.drone.yml

```diff
@@ -99,6 +99,18 @@ pipeline:
       matrix:
         TEST_SUITE: phpunit
 
+  scrutinizer:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+      - PHP_VERSION=${PHP_VERSION}
```

Comment:
shouldn't be needed - but I might be missing something

Reply:
true, I've just copied it from `phpunit-tests:` now wondering why we are setting it there